org.quantil.qprov.core/src/main/java/org/quantil/qprov/core/model/entities/GateCharacteristics.java

```diff
@@ -60,7 +51,7 @@ public class GateCharacteristics implements Comparable<GateCharacteristics> {
     private BigDecimal gateTime;
 
     @Column(precision = Constants.BIG_DECIMAL_PRECISION, scale = Constants.BIG_DECIMAL_SCALE)
-    private BigDecimal gateFidelity;
+    private BigDecimal gateErrorRate;
```

Comment:
If we rename it the IBMQ provider must calculate the error rate from the fidelity? Or was it just a wrong name until now?

Reply:
It is only a wrong name. :)